src/vcpkg/commands.hash.cpp

```diff
@@ -20,7 +20,8 @@ namespace vcpkg::Commands::Hash
     {
         (void)args.parse_arguments(COMMAND_STRUCTURE);
 
-        const Path file_to_hash = args.command_arguments[0];
+        Path file_to_hash = args.command_arguments[0];
+        if (file_to_hash.is_relative()) file_to_hash = (paths.original_cwd / file_to_hash).lexically_normal();
```

Comment:
```suggestion
        auto file_to_hash = (paths.original_cwd / args.command_arguments[0]).lexically_normal();
```

`Path::operator/` does nothing if the right hand side is absolute, so no need to guard against it.